internal/project/framework/internals/async/async.go

```diff
@@ -1,12 +1,13 @@
-{{ .Copyright }}
+/**#bean*/ /*#bean.replace({{ .Copyright }})**/
 // Safe way to execute `go routine` without crashing the parent process while having a `panic`.
 package async
 
 import (
 	"fmt"
 	"reflect"
 
-	"{{ .PkgPath }}/framework/internals/global"
+	/**#bean*/
+	"demo/framework/internals/global" /*#bean.replace("{{ .PkgPath }}/framework/internals/global")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines